app/ios/AriesBifold.xcodeproj/project.pbxproj

```diff
@@ -537,7 +537,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = AriesBifold/AriesBifold.entitlements;
 				CURRENT_PROJECT_VERSION = 444;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = L796QSLV3E;
```

Comment:
I don't think this file is meant to have changes checked in. Able to remove it from the PR?